### DISCUSSION TITLE
default option for qselect

### DISCUSSION THREAD

**Discussion by albukzek** [quasarframework/quasar#13859] [General - Components / Directives / etc]
Hi all.
How to set the default option for qselect ?

**Comment by MilosPaunovic** [comment]
[https://codepen.io/milospaunovic/pen/eYMpYev?editors=101](https://codepen.io/milospaunovic/pen/eYMpYev?editors=101)

**Reply by albukzek** [reply]
> https://codepen.io/milospaunovic/pen/eYMpYev?editors=101

So simple. Thanks :)
